query/src/lib.rs

```diff
@@ -68,6 +69,12 @@ impl Variable {
     }
 }
 
+impl fmt::Debug for Variable {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "{}", self.0)
```

Comment:
There's no requirement that `Debug` output follow any particular format, so no need for EDN — we could just as easily do `write!(f, "var({})", self.0)` if we wanted to be unambiguous.